src/Files.App/Filesystem/Security/AccessMaskFlags.cs

```diff
@@ -0,0 +1,131 @@
+﻿using System;
+
+namespace Files.App.Filesystem.Security
+{
+	/// <summary>
+	/// Represents access mask flags of an ACE.
+	/// </summary>
+	[Flags]
+	public enum AccessMaskFlags
+	{
+		/// <summary>
+		/// No access mask flags
+		/// </summary>
+		NULL = 0,
+
+		/// <summary>
+		/// Read data
+		/// </summary>
+		ReadData = 1,
+
+		/// <summary>
+		/// List data
+		/// </summary>
+		ListDirectory = 1,
+
+		/// <summary>
+		/// Write data
+		/// </summary>
+		WriteData = 2,
+
+		/// <summary>
+		/// Create files
+		/// </summary>
+		CreateFiles = 2,
+
+		/// <summary>
+		/// Append data
+		/// </summary>
+		AppendData = 4,
+
+		/// <summary>
+		/// Create folders
+		/// </summary>
+		CreateDirectories = 4,
+
+		/// <summary>
+		/// Read extended attributes
+		/// </summary>
+		ReadExtendedAttributes = 8,
```

Comment:
I'm no expert in security and mask access, but having flags with different meaning sharing the same values seems off... Is this normal?

Reply:
Yes. You can refer to ACCESS_MASK in MSDN.